src/commit.ts

```diff
@@ -27,6 +27,20 @@ async function commitHeadersSinceVersion(version: string): Promise<string[]> {
   return commitsSinceLastVersionOutput.stdout.trim().split('\n');
 }
 
+function getTypeAndScope(prefix: string): { type: string, scope?: string } {
+  const matches = /^([^(]*)(?:\(([^)]+)\))?:/.exec(prefix);
+  if (!matches) {
+    // Malformed commit prefix. Try our best.
+    return { 
+      type: prefix.split(':')[0] || ''
```

Comment:
I've kept this logic from previous behaviour in case some existing user is reliant on it.